cypress/e2e/pagination.cy.js

```diff
@@ -1,33 +1,132 @@
 describe("Pagination", () => {
   beforeEach(() => {
-      cy.setCookie("Other", "other");
-      cy.setCookie("XSRF-TOKEN", "abcde");
-      cy.visit("/supervision/workflow/1");
+    cy.setCookie("Other", "other");
+    cy.setCookie("XSRF-TOKEN", "abcde");
+    cy.visit("/supervision/workflow/1");
   });
 
-  it("shows correct number of total tasks", () => {
-    cy.get("#pagination-label > .flex-container > .moj-pagination__results > :nth-child(1)").should('contain', '1')
-    cy.get("#pagination-label > .flex-container > .moj-pagination__results > :nth-child(2)").should('contain', '10')
-    cy.get("#pagination-label > .flex-container > .moj-pagination__results > :nth-child(3)").should('contain', '10')
-  })
+  describe("First page, ellipses and final page", () => {
+    beforeEach(() => {
+      cy.get("#display-rows").select('25')
+      cy.get("#display-rows").should('have.value', '25')
+    })
 
- it("disabled previous button while on top page one", () => {
-    cy.get("#pagination-label > .flex-container > .moj-pagination__list > .moj-pagination__item--prev > .moj-pagination__link").should('be.disabled')
-  })
+    it("will not show previous arrow on page 1 but will show it on other pages", () => {
+      cy.get("#display-rows").select('25')
+      cy.get("#display-rows").should('have.value', '25')
+      cy.get("#previous-page-pagination-link").should('not.exist')
+      cy.get("#next-page-pagination-link").click()
+      cy.get("#previous-page-pagination-link").should('be.visible', 'Previous')
+    })
 
-  it("can select 25 from task view value dropdown", () => {
-    cy.get("#display-rows").select('25')
-    cy.get("#display-rows").should('have.value', '25')
-  })
+    it("shows next button apart from on last page", () => {
+      cy.get("#next-page-pagination-link").should('be.visible', 'Next')
+      cy.get(".govuk-\\!-padding-right-2 > #pagination-label > .flex-container > #numbered-pages > .govuk-pagination__list > #final-page-pagination-link > .govuk-link").click()
```

Comment:
These selectors don't need to be this long. Ids (`#final-page-pagination-link`) should be unique on a page, so you can just use `#final-page-pagination-link button` rather than all the extra info to uniquely identify the thing to click.

That said. It looks like the pagination is repeated twice, but uses the same template. So worth looking at making it so top of page pagination has different IDs to bottom of page pagination otherwise you'll get collisions.